bin-ranges-transfer/src/main/java/uk/gov/pay/java_lambdas/bin_ranges_transfer/util/Utils.java

```diff
@@ -45,15 +46,29 @@ public static String extractDateString(String fileName) throws DateTimeParseExce
 
         if (matcher.find()) {
             String dateString = matcher.group();
-            logger.debug("date pattern found in file name: {}", fileName);
+            logger.debug("Date pattern [{}] found in file name: {}", dateString, fileName);
             
             DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyyMMdd");
             LocalDate date = LocalDate.parse(dateString, formatter);
             
-            logger.debug("date validated: {}", date);
+            logger.debug("Date validated: {}", date);
             return date.toString();
         } else {
-            throw new IllegalArgumentException(String.format("no date string found in file name: %s", fileName));
+            throw new IllegalArgumentException(String.format("No date string found in file name: %s", fileName));
+        }
+    }
+    
+    public static Version extractVersion(String fileName) throws IllegalArgumentException {
+        String versionPattern = "V\\d{2}";
+        Pattern pattern = Pattern.compile(versionPattern);
```

Comment:
This can be a constant (`Pattern`s are threadsafe).

Reply:
done